eldritch/events.py

```diff
@@ -1199,45 +1199,27 @@ def __init__(self, character):
     super().__init__([ForceMovement(character, "Lost"), LoseTurn(character)], character)
 
 
-class BlessCurse(Event):
-
+class BlessCurse(Sequence):
   def __init__(self, character, positive):
-    super().__init__()
-    self.character = character
-    self.adjustment = 1 if positive else -1
-    self.change = None
-
-  def resolve(self, state):
-    old_val = self.character.bless_curse
-    new_val = min(max(old_val + self.adjustment, -1), 1)
-    self.character.bless_curse = new_val
-    self.change = new_val - old_val
-    if new_val != 0:
-      self.character.bless_curse_start = state.turn_number + 2
+    if positive:
+      self.card = "Blessing"
```

Comment:
I think my original vision was that if you were checking whether an event was a Bless or Curse, you could check for the event.card, but then I just made them their own class, because `if isinstance(event, Bless)` is nicer than `if isinstance(event, BlessCurse) and event.card == "Blessing"`. 

Removing the attribute.